File: domain/src/main/java/no/nav/melosys/domain/dokument/soap/inntekt/ArbeidsforholdFrilanser.java
package no.nav.melosys.domain.dokument.soap.inntekt;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;

@XmlAccessorType(XmlAccessType.FIELD)
public class ArbeidsforholdFrilanser {

    public Periode frilansPeriode;

    public String yrke; //"http://nav.no/kodeverk/Kodeverk/Yrker"

    public Periode getFrilansPeriode() {
        return frilansPeriode;
    }

    public void setFrilansPeriode(Periode frilansPeriode) {
        this.frilansPeriode = frilansPeriode;
    }

    public String getYrke() {
        return yrke;
    }

    public void setYrke(String yrke) {
        this.yrke = yrke;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/dokument/inntekt/ArbeidsforholdFrilanser.kt
package no.nav.melosys.domain.dokument.inntekt

data class ArbeidsforholdFrilanser(
    val frilansPeriode: Periode? = null,
    val yrke: String? = null //"http://nav.no/kodeverk/Kodeverk/Yrker"
)
